behavior_tests/src/cmp-cmds-linker-entry-src-files/Makefile

```diff
@@ -0,0 +1,55 @@
+#===============================================================================
+# User Options
+#===============================================================================
+
+# Compiler can be set below, or via environment variable
+CC        = nvcc
+OPTIMIZE  = yes
+DEBUG     = no
+ARCH      = sm_60
+
+#===============================================================================
+# Program name & source code list
+#===============================================================================
+
+program = main
+
+source = one.cu two.cu three.c four.c 
+
+obj = $(source:.cu=.o)
+
+#===============================================================================
+# Sets Flags
+#===============================================================================
+
+# Standard Flags
+CFLAGS := $(EXTRA_CFLAGS) -std=c++14 -Xcompiler -Wall -arch=$(ARCH)
+
+# Linker Flags
+LDFLAGS = 
+
+# Debug Flags
+ifeq ($(DEBUG),yes)
+  CFLAGS += -g -DDEBUG
+  LDFLAGS  += -g
+endif
+
+# Optimization Flags
+ifeq ($(OPTIMIZE),yes)
+  CFLAGS += -O3
+endif
+#===============================================================================
+# Targets to Build
+#===============================================================================
+
+$(program): $(obj) 
+	$(CC) $(CFLAGS) $(obj) -o $@ $(LDFLAGS)
```

Comment:
in link entry, there is no src file here, it all .o files.  So need update a little as test for #1112.

Reply:
https://github.com/oneapi-src/SYCLomatic-test/pull/413/files#diff-6858f4e8178f10093a4fd4eb0594f421bdaeab157a425cd70a39ac5fd3c337caR17-R19

I think the variable `obj` has the source files `one.o` , `two.o` and `three.c` and `four.c` included in these lines. It is generating the linker entry with the source files. 

Can you reconfirm please.

Reply:
Suggest adding explicit linker command in the Makefile which includes source files, so that it is friendly to read.

Reply:
Done.